server/data/locationsApiClient.ts

```diff
@@ -6,7 +6,46 @@ export interface Location {
   prisonId: string
   code: string
   pathHierarchy: string
-  locationType: string
+  locationType:
+    | 'WING'
```

Comment:
Typescript types must be defined statically, so we can't fetch these types from API or DB.

I got these values from https://github.com/ministryofjustice/hmpps-locations-inside-prison-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/LocationType.kt, I might make a script that automatically updates our types from the API files, so that we can stay up to date.

I can make it a type with `type LocationType = ...`, which I will do.